### PR TITLE
Cleanup Native Image Tests

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -11,7 +11,10 @@ if (properties.containsKey('android.injected.invoked.from.ide') ||
   include ':android-test'
 }
 
-include ':native-image-tests'
+//if (!properties.containsKey('android.injected.invoked.from.ide')) {
```
Comment (yschimke) [inline settings.gradle:14]: Reverting this.

File: native-image-tests/build.gradle [hunk 2]
```diff
@@ -39,29 +39,6 @@ animalsniffer {
...
 
-def graalHome = "/Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.2.0/Contents/Home"
-def graalvmResources = project.file("$buildDir/graalvm/resources")
-def graalvmConfig = project.file("$graalvmResources/META-INF/native-image")
-graalvmConfig.mkdirs()
-
-sourceSets {
-  main.resources.srcDirs += graalvmResources
-
-  main.java.srcDirs += project.file("../okhttp-logging-interceptor/src/test/java")
```
Comment (yschimke) [inline native-image-tests/build.gradle:50]: Will reenable with each batch of tests

Comment (yschimke) [reply native-image-tests/build.gradle:50]: https://github.com/square/okhttp/pull/6356
https://github.com/square/okhttp/pull/6355

**Review by swankjesse — APPROVED** [review]

File: native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt [hunk 3]
```diff
@@ -15,12 +15,37 @@
...
+    client.newCall(Request.Builder().url(server.url("/")).build()).execute().use {
+      assertThat(it.body!!.string()).isEqualTo("abc")
+    }
+  }
+
+  @Test
+  fun testExternalSite() {
+    val client = clientRule.newClient()
+
+    client.newCall(Request.Builder().url("https://google.com/robots.txt").build()).execute().use {
```
Comment (swankjesse) [inline native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt:47]: will probably want to get rid of this for running on CI

Comment (yschimke) [reply native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt:47]: What is the junit5 way of doing this?  I'd like a way to say run this assuming a network and non-hermetic tests allowed.

Comment (swankjesse) [reply native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt:47]: Throw `org.opentest4j.TestAbortedException`, perhaps by calling `Assumptions.assumeTrue(false)`

Comment (swankjesse) [reply native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt:47]: Throw `org.opentest4j.TestAbortedException`, perhaps by calling `org.junit.jupiter.api.Assumptions.assumeTrue(false)`

Comment (yschimke) [reply native-image-tests/src/main/kotlin/okhttp3/SampleTest.kt:47]: I was thinking https://junit.org/junit5/docs/current/user-guide/#writing-tests-tagging-and-filtering

@Tag("remote")
@Tag("fast")

**Pull request merged**